AI #2350: Add Conditions to the spec (#2351)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/conditions/includesmultiplepricingcategories.md

# Includes Multiple Pricing Categories

The Includes Multiple Pricing Categories condition represents a verifiable state where the [operating model](#glossary:operating-model) includes more than one pricing category across all SKUs.

## Requirements

IncludesMultiplePricingCategories MUST adhere to the following requirements:

> * IncludesMultiplePricingCategories MUST evaluate to true when the operating model supports more than one pricing category (e.g., On-Demand, Reservation, Spot).

Author: ```suggestion
* IncludesMultiplePricingCategories MUST evaluate to true when the operating model supports more than one pricing category (e.g., Standard, Dynamic, Committed).
```

> * IncludesMultiplePricingCategories MUST evaluate to false when the operating model supports only a single pricing category.

Reviewer: I don’t think we need to prevent providers from exposing a value when only a single value exists.

North star (but I expect pushback 🙂): I would prefer that providers always expose the value explicitly, regardless of whether there is one or multiple values, to avoid practitioners having to infer the value from the documentation.

```suggestion
```

Author: I don't disagree, but I think we should phrase this for now in a way that matches what the spec currently says.  To that end, what do you think about:

```suggestion
* IncludesMultiplePricingCategories MAY evaluate to false when the operating model supports only a single pricing category.
```
